### PR TITLE
Finish mobile version PR

### PR DESCRIPTION
**Pull request by HunterX-7**
In this Pull Request, I:
> Create the finishmobile branch.

> Followed GitHub flow.

> Use HTML&CSS Linter.

> Write meaningful commit in the most important parts of the process.

> As requested use CSS Grid and Flexbox combined to layout the elements.

> Create the work section.

-  Use ul and li tags for categories in the "Work" section. Listing HTML, CSS, and JavaScript.

-  Add the images for the projects.

> Create the About myself section

-  Use ul and li for categories in the "About myself" section.

-  Add the images for the skills icons.

-  Add the button interactions (hover, active, etc.).

> Finally I stick to the design in Figma Template #3 as much as possible.

Work section template
About me section template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,5 +33,220 @@ <h1 class="title1">Hey there. I'm Francisco<br><span class="title2">I'm a softwa
...
+                <ul class="skills2">
+                    <li class="skills3">Codekit</li>
+                    <li class="skills3">GitHub</li>
+                    <li class="skills3">Codepen</li>
+                    <li class="skills3">Gitlab</li>
+                    <li class="skills3">Terminal</li>
+                </ul>
+            </div>
+        </div>
+    </section>
```
Comment (Santiago220991) [inline index.html]: I think you did a great job here, however, there are some things to improve here. :+1: :smiley: :smiley: 
- This block of code should be separated into two sections. One for "Works" and another one for "About me". Kindly wrap these sections inside `section` tags. Any other `section` tag should be removed. Every work or skill card should be inside a `div` tag, not a section tag. 
image

- The images in the about section should be added as background images using CSS. Decorations are always background images, not `img` tags. :nerd_face: :nerd_face:  Kindly change this. 
image

File: style.css [hunk 2]
```diff
@@ -78,6 +119,50 @@ p {
...
+.container2 {
+  height: 352px;
+  margin-left: 20px;
+  margin-right: 20px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: grid;
+  grid-template-rows: 184px 32px 136px;
+}
```
Comment (Santiago220991) [inline style.css]: Kindly make sure that you are using flexbox to organize the elements inside the skills cards. This is part of the requirements of the project. :eyes: :eyes: 
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @HunterX-7, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Good PR summary.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Good commit messages.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HunterX-7** [comment]
In this Pull Request, I made the following requested changes:

- Remove unnecessary section tags and add a comment for the work section and the about me section for clarity.
- Remove the img tags in the about me section and replace them as background images.
- Organize the skills cards using flexbox.

About me section template

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @HunterX-7,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

- [OPTIONAL] Kindly make your social media handle a link you can click.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**